src/aot/aot_main.cpp

```diff
@@ -18,6 +18,7 @@ void usage(std::string_view filename)
   std::cerr << "USAGE: " << filename << " [options]" << std::endl;
   std::cerr << std::endl;
   std::cerr << "OPTIONS:" << std::endl;
+  std::cerr << "    -d STAGE       debug info for various stages of bpftrace execution" << std::endl;
```

Comment:
Nit: probably better to place this down right below `-v` line. B/c debug is less likely to be used regularly

Reply:
Done, thanks.